### PR TITLE
Emp 18019

### PR DESCRIPTION
**Pull request by udayasri**

### CODE REVIEW

File: Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift [hunk 1]
```diff
@@ -348,75 +348,42 @@ extension ServerSideAdService {
...
                                             self.alreadySentadTrackingUrls.append( firstQuartileUrls )
                                         }
                                     }
     
                                 } else if (timeInMil / 10 * 10) == (clipMidpoint / 10 * 10) {
                                     // Send clipMidpoint tracking events
-                                    
                                     if let midpointUrls = clip.trackingEvents?.midpoint {
                                         if !self.alreadySentadTrackingUrls.contains(midpointUrls) {
+                                            print("midpoint")
```
Comment (delarge77) [inline Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift]: Better to remove print here too

File: Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift [hunk 2]
```diff
@@ -348,75 +348,42 @@ extension ServerSideAdService {
...
                                             self.alreadySentadTrackingUrls.append( midpointUrls )
                                         }
                                     }
                                     
                                 } else if (timeInMil / 10 * 10) == (clipThirdQuartile / 10 * 10)  {
                                     // Send thirdQuartile tracking events
-                                    
                                     if let thirdQuartileUrls = clip.trackingEvents?.thirdQuartile {
                                         if !self.alreadySentadTrackingUrls.contains(thirdQuartileUrls) {
+                                            print("thirdQuartile")
```
Comment (delarge77) [inline Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift]: and here

File: Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift [hunk 3]
```diff
@@ -348,75 +348,42 @@ extension ServerSideAdService {
...
                                             self.alreadySentadTrackingUrls.append( thirdQuartileUrls )
                                         }
                                     }
                                     
                                 } else if (timeInMil / 10 * 10) == (end / 10 * 10)  {
-                                    
                                     // Send complete tracking events
                                     if let completeUrls = clip.trackingEvents?.complete {
                                         if !self.alreadySentadTrackingUrls.contains(completeUrls) {
+                                            print("complete")
```
Comment (delarge77) [inline Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift]: and here

**Review by delarge77 — APPROVED** [review]
Just take a look on the comments, and we are ready to go .

File: Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift [hunk 4]
```diff
@@ -348,75 +348,42 @@ extension ServerSideAdService {
...
-                                }
-                                
-                                else if (timeInMil / 10 * 10) == (clipFirstQuartile / 10 * 10)  {
+
+                                if (timeInMil / 10 * 10) == (clipFirstQuartile / 10 * 10)  {
                                     // Send firstQuartile tracking events
-                                    
                                     if let firstQuartileUrls = clip.trackingEvents?.firstQuartile {
                                         if !self.alreadySentadTrackingUrls.contains(firstQuartileUrls) {
+                                            print("firstQuartile")
```
[the remark this answers is not in the document]
Comment (udayasri) [reply Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift]: 👍

File: Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift (hunk 1 above)
Comment (udayasri) [reply Sources/iOSClientExposurePlayback/Services/AdService/ServerSideAdService.swift]: 👍

**Pull request merged**